src/middlewares/auth/utils/isAuthHandlerByEnrolementId.js

```
@@ -0,0 +1,19 @@
+const UnauthorizedError = require("../../../exceptions/UnauthorizedError");
+const { Enrolment, Member } = require("../../../models");
+
+const isAuthHandlerByEnrolementId = async (req, res) => {
+  const enrolmentId = req.params.enrolmentId;
+  //console.log("enrolmentId from body:", enrolmentId);
+  const enrolment = await Enrolment.findById(enrolmentId);
+  if (!enrolment) return false;
```

IsAuthHandler should throw Error (instead of returning false).